### DISCUSSION TITLE
NES Save Dump

### DISCUSSION THREAD

**Discussion by Coaleb2** [sanni/cartreader#66] [Need help]
You mention we can dump nes saves in your readme. I do not see a way how when I try to dump the save. What are the steps to dumping an nes save?

**Comment by sanni** [comment]
The "Read RAM" option should dump the SRAM chip containing the save data to the SD card. I can't try it myself though since I currently do not own a NES game with save. You might get a more detailed answer in the Arduino forum thread: https://forum.arduino.cc/index.php?topic=158974.new#new
